packages/ui/src/components/organisms/DynamicForm/components/RSJVInputAdaters/RadioInputAdapter/RadioInputAdapter.tsx

```diff
@@ -0,0 +1,35 @@
+import { Label } from '@/components/atoms';
+import { RadioGroup, RadioGroupItem } from '@/components/atoms/RadioGroup';
+import { RJSFInputAdapter } from '@/components/organisms/DynamicForm/components/RSJVInputAdaters/types';
+import { TOneOfItem } from '@/components/organisms/DynamicForm/types/one-of';
+import { useMemo } from 'react';
+
+export const RadioInputAdapter: RJSFInputAdapter<string> = ({
+  id,
+  schema,
+  formData,
+  disabled,
+  onBlur,
+  onChange,
+}) => {
+  const options = useMemo(
+    () => (schema?.oneOf as TOneOfItem[])?.map(item => ({ label: item.title, value: item.const })),
+    [schema],
+  );
+
+  return options?.length ? (
```

Comment:
`0` will not render if you use `!!`. 🙂  Consider an early return for readability.
```suggestion
  return !!options?.length ? (
```

Reply:
P3